galaxy_ng/tests/integration/api/test_rbac_roles.py

```diff
@@ -340,6 +329,71 @@ def test_global_role_actions(role):
     assert failures == []
 
 
+@pytest.mark.role_rbac
+@pytest.mark.standalone_only
+def test_object_role_actions():
+    registry = ReusableContainerRegistry(gen_string())
+    registry_pk = registry.get_registry()["pk"]
+
+    users_and_groups = {}
+    col_groups = []
+    ee_groups = []
+
+    # Create user/group for each role
+    # Populate collection/ee groups for object assignment
+    for role in ROLES_TO_TEST:
```

Comment:
We don't need to test all the roles. The UI only lets you assign object permissions for roles that only have permissions on the object. Take a look at the UI for namespace and EE owners and see which roles are allowed, and let's stick to just testing those.